rollup.config.ts

```diff
@@ -17,7 +17,7 @@ const plugins = [
 ];
 
 export default defineConfig([{
-    input: "lib/index.standalone.ts",
+    input: "lib/index.ts",
```

Comment:
Since the `input` is now the same for all files, we can put it in a variable so it can be re-used.